packages/crypto/src/sha1.rs

```diff
@@ -1,21 +1,8 @@
 use sha1::{Digest, Sha1};
 
-use crate::errors::{CryptoError, CryptoResult};
-
-/// Max count of a inputs for sha1.
-/// A limit is set to prevent malicious excessive input.
-/// Now, we limit ourselves to only small sizes for use cases in Uuid.
-pub const INPUTS_MAX_CNT: usize = 4;
-
-/// Max length of a input for sha1
-/// After executing the crypto bench according to INPUT_MAX_LEN,
-/// the gas factor is determined based on the result.
-/// If you modify this value, you need to adjust the gas factor.
-pub const INPUT_MAX_LEN: usize = 80;
```

Comment:
when I try run `cargo bench`, that failed.

```
error[E0432]: unresolved import `cosmwasm_crypto::INPUT_MAX_LEN`
  --> packages/crypto/benches/main.rs:16:21
   |
16 |     sha1_calculate, INPUT_MAX_LEN,
   |                     ^^^^^^^^^^^^^ no `INPUT_MAX_LEN` in the root

For more information about this error, try `rustc --explain E0432`.
```

and I think we need change this comment also. is it okay?
`// 270 ns in crypto benchmarks when INPUT_MAX_LEN=80` => `MAX_LENGTH_SHA1_MESSAGE`

https://github.com/line/cosmwasm/blob/9c0977858666287eb859c4a663d33530eae6a311/packages/vm/src/environment.rs#L59

Reply:
thanks! fixed.